pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -39,12 +45,35 @@ func (pl *TaintToleration) Name() string {
 }
 
 // Filter invoked at the filter extension point.
-func (pl *TaintToleration) Filter(_ *framework.CycleState, pod *v1.Pod, nodeInfo *nodeinfo.NodeInfo) *framework.Status {
+func (pl *TaintToleration) Filter(state *framework.CycleState, pod *v1.Pod, nodeInfo *nodeinfo.NodeInfo) *framework.Status {
```

Comment:
I think it's worth adding a comment saying that this predicate doesn't use the metadata. Same for NormalizeScore below.

Reply:
done.